cmd/prepare-storage-msg_test.go

```diff
@@ -18,9 +18,38 @@ package cmd
 
 import (
 	"net/url"
+	"reflect"
 	"testing"
 )
 
+func TestHealEndpoint(t *testing.T) {
+	tls := true
+	hURL := getHealEndpoint(tls, &url.URL{
+		Scheme: "http",
+		Host:   "localhost:9000",
+	})
+	sHURL := &url.URL{
+		Scheme: "https",
+		Host:   "localhost:9000",
+	}
+	if !reflect.DeepEqual(hURL, sHURL) {
+		t.Fatalf("Expected %#v, but got %#v", sHURL, hURL)
+	}
+
+	tls = false
+	hURL = getHealEndpoint(tls, &url.URL{
+		Scheme: "https",
+		Host:   "localhost:9000",
+	})
+	sHURL = &url.URL{
+		Scheme: "http",
+		Host:   "localhost:9000",
+	}
+	if !reflect.DeepEqual(hURL, sHURL) {
+		t.Fatalf("Expected %#v, but got %#v", sHURL, hURL)
+	}
```

Comment:
I avoided the globalHost case on purpose, so that we don't add more racy tests until @hackintoshrao finishes his work on saving global states.